includes/class-ssp-stats.php

```diff
@@ -297,6 +301,8 @@ public function track_download ( $file = '', $episode = 0, $referrer = '' ) {
 			$referrer = 'overcast';
 		} elseif ( stripos( $user_agent, 'Pocket Casts' ) !== false ) {
 			$referrer = 'pocketcasts';
+		} elseif( stripos( $user_agent, 'Android' ) !== false || stripos( $user_agent, 'PodcastAddict' ) !== false || stripos( $user_agent, 'Player FM' ) !== false || stripos( $user_agent, 'Google-Play' ) !== false ){
```

Comment:
@TheCraigHewitt @JTrial based on the above checks, I feel like we shouldn't just be looking for a result of 'is android' but we should rather be counting android, podcast addict, player fm and google play as separate services. for example iTunes is a service, which can be used on an iOS device or an OSX device, similarly PocketCasts is both an iOS and Android app.

Reply:
@jonathanbossenger I fully get where you're coming from. I've separated the newly added services into separate items to ensure they're counted individually. The revision has been added below. I hope this is okay?